src/app/service/poap/config/ModifyPOAP.ts

```diff
@@ -1,34 +1,46 @@
-import { GuildMember, Message, MessageEmbedOptions, MessageReaction, Role } from 'discord.js';
-import ValidationError from '../../errors/ValidationError';
+import {
+	GuildMember,
+	Message,
+	MessageEmbedOptions,
+	MessageOptions,
+	MessageReaction,
+	Role,
+	TextChannel,
+} from 'discord.js';
+import ValidationError from '../../../errors/ValidationError';
 import { Collection } from '@discordjs/collection';
 import {
 	BulkWriteError,
-	Collection as MongoCollection,
+	Collection as MongoCollection, CollectionInsertManyOptions,
```

Comment:
Will you put `CollectionInsertManyOptions` on its own line?

Reply:
Sure but FYI I'll need to create a new PR since this is the 2nd stage of the review